projects/optic/src/__tests__/integration/api-add.test.ts

```diff
@@ -85,7 +85,7 @@ describe('optic api add', () => {
       }
       const { combined, code } = await runOptic(
         workspace,
-        'api add ./spec.yml --standard blah'
```

Comment:
I removed this entirely because the chances someone knows the id of their standard was basically 0 anyway. Web and or optic yaml makes more sense